.vitepress/sidebars/develop.ts

```diff
@@ -46,6 +46,10 @@ export default [
       {
         text: "develop.gettingStarted.launchGame",
         link: "/develop/getting-started/launching-the-game"
+      },
+      {
+        text: "develop.gettingStarted.usingIde",
```

Comment:
May also be worth renaming the translation key and changing the URL to "ide-tips-and-tricks" or something similar.